exporter/awsemfexporter/emf_exporter_test.go

```diff
@@ -192,10 +194,15 @@ func TestConsumeMetricsWithLogGroupStreamConfig(t *testing.T) {
 	})
 	require.Error(t, exp.pushMetricsData(ctx, md))
 	require.NoError(t, exp.shutdown(ctx))
-	pusherMap, ok := exp.pusherMap[cwlogs.StreamKey{
+	streamKey := &cwlogs.StreamKey{
 		LogGroupName:  expCfg.LogGroupName,
 		LogStreamName: expCfg.LogStreamName,
-	}]
+		Entity: &cloudwatchlogs.Entity{KeyAttributes: map[string]*string{
+			entityType: aws.String(service),
```

Comment:
Are you referring to type in https://w.amazon.com/bin/view/AWS/AIO/Specs/O2F#H2.1.1TypeService? why the name is entityType? I see it should be `Type` https://github.com/aws/private-amazon-cloudwatch-agent-staging/blob/main-compass/extension/entitystore/extension.go#L182

Reply:
entityType is the variable name for `Type` https://github.com/amazon-contributing/opentelemetry-collector-contrib/pull/233/files#diff-af3bf0e3a5a889da31842d592ffeecc4ef5d3e5b058af7ca500ed44e321bca0cR44